### PR TITLE
Add eastl/3.15.00 recipe

### PR DESCRIPTION
**Pull request by madebr**
Specify library name and version:  **eastl/3.15.00**

Depends on #290 

https://github.com/electronicarts/EASTL

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [X] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [X] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by lasote** [comment]
Fixed eabase and relaunched.

**Comment by uilianries** [comment]
@madebr take a look https://github.com/madebr/conan-center-index/pull/1

File: recipes/eastl/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
+        self.copy("3RDPARTYLICENSES.TXT", src=self._source_subfolder, dst="licenses")
+        cmake = self._configure_cmake()
+        cmake.install()
+
+    def package_info(self):
+        self.cpp_info.name = "EASTL"
+        self.cpp_info.libs = ["EASTL"]
+        if self.settings.os in ("Android", "Linux", "Macos", "watchOS", "tvOS"):
+            self.cpp_info.libs.append("pthread")
```
Comment (uilianries) [inline recipes/eastl/all/conanfile.py]:
```suggestion
            self.cpp_info.system_libs.append("pthread")
```

Comment (madebr) [reply recipes/eastl/all/conanfile.py]: Ok, no force push this time

Comment (uilianries) [reply recipes/eastl/all/conanfile.py]: Nice! :sweat_smile:

**Pull request merged**